src/main.js

```diff
@@ -35,112 +35,118 @@ startButton.addEventListener('click', startCountDown);
 logButton.addEventListener('click', logActivity);
 createNewActivityBtn.addEventListener('click', createNewActivity);
 
-function changeColor(button1, category1, button2, category2, button3, category3) {
-    if (button2 == undefined) {
-        button1.classList.add(category1);
-    } else if (button3 == undefined) {
-        button1.classList.add(category1);
-        button2.classList.remove(category2);
-    } else {
-        button1.classList.add(category1);
-        button2.classList.remove(category2);
-        button3.classList.remove(category3);
-    }
+function addClassProperty(button1, category1, button2, category2) {
+  if (!button2) {
+    button1.classList.add(category1);
+  } else {
+    button1.classList.add(category1);
+    button2.classList.add(category2);
+  }
+}
+
+function removeClassProperty(button1, category1, button2, category2) {
+  if (!button2) {
+    button1.classList.remove(category1);
+  } else {
+    button1.classList.remove(category1);
+    button2.classList.remove(category2);
+  }
 }
 
 function changeStudyColor() {
-    changeColor(studyButton, 'study-active', meditateButton, 'meditate-active', exerciseButton, 'exercise-active');
-    isCatChosen(studyButton, meditateButton, exerciseButton);
+  addClassProperty(studyButton, 'study-active');
+  removeClassProperty(meditateButton, 'meditate-active', exerciseButton, 'exercise-active')
+  isCatChosen(studyButton, meditateButton, exerciseButton);
   }
 
 function changeMeditateColor() {
-    changeColor(meditateButton, 'meditate-active', studyButton, 'study-active', exerciseButton, 'exercise-active');
-    isCatChosen(studyButton, meditateButton, exerciseButton);
+  addClassProperty(meditateButton, 'meditate-active');
+  removeClassProperty(studyButton, 'study-active', exerciseButton, 'exercise-active');
+  isCatChosen(studyButton, meditateButton, exerciseButton);
   }
 
 function changeExerciseColor() {
-    changeColor(exerciseButton, 'exercise-active', studyButton, 'study-active', meditateButton, 'meditate-active');
-    isCatChosen(studyButton, meditateButton, exerciseButton);
+  addClassProperty(exerciseButton, 'exercise-active');
+  removeClassProperty(studyButton, 'study-active', meditateButton, 'meditate-active');
+  isCatChosen(studyButton, meditateButton, exerciseButton);
   }
 
 function limitTimeInput(timeInput, num) {
   preventInvalids(event, timeInput);
   timeInput.value = parseInt(timeInput.value);
   if (timeInput.value > num) {
-      timeInput.value = num;
+    timeInput.value = num;
   }
 }
 
 function preventInvalids(event, inputField) {
```

Comment:
this is great! Just for fun check out regex later. really cool option for character validation and such.